data/src/main/java/com/alexandruvalentinconstantin/data/NewsRepository.java

```diff
@@ -0,0 +1,29 @@
+package com.alexandruvalentinconstantin.data;
+
+import com.alexandruvalentinconstantin.data.feature.news.local.ArticleEntity;
+import com.alexandruvalentinconstantin.data.feature.news.model.Article;
+
+import java.util.List;
+
+import io.reactivex.Completable;
+import io.reactivex.Single;
+import io.reactivex.annotations.NonNull;
+
+public interface NewsRepository {
+
+    @NonNull
+    Single<List<Article>> getNewsArticles();
```

Comment:
there seems to be a duplicated method for getting the Single<List<Article>>. Let's keep only one of these.